uportal-war/src/main/webapp/WEB-INF/jsp/BackgroundPreference/viewBackgroundPreference.jsp

```diff
@@ -139,8 +139,8 @@
         </a>
         <c:forEach var="image" items="${thumbnailImages}" varStatus="status">
             <a href="#">
-                <img src="${image}" />
-                <span class="caption">Background ${status.index + 1}</span>
+                <img src="${image}" alt="" role="presentation"/>
```

Comment:
@drewwills I made it more specific - the alt text attribute itself was added (it has to be present to pass ADA checks).

I've tested the change without the preference - it just shows the picture without a caption and doesn't appear to cause any crashes.